Runtime error on armv7

Get this when trying to start the Tracer:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x4 pc=0x123e4]
goroutine 1 [running]:
xxxx/vendor/github.com/DataDog/datadog-go/statsd.(*Client).writeMetricUnsafe(0xd282a0, 0x1, 0x0, 0x0, 0xcf67e0, 0x4, 0x6, 0x57161e, 0x16, 0x0, ...)
        xxxxxxxxxxx/vendor/github.com/DataDog/datadog-go/statsd/statsd.go:442 +0x70
xxxx/vendor/github.com/DataDog/datadog-go/statsd.(*Client).addMetric(0xd282a0, 0x1, 0x0, 0x0, 0xcf67e0, 0x4, 0x6, 0x57161e, 0x16, 0x0, ...)
        xxxxxxxxxxx/vendor/github.com/DataDog/datadog-go/statsd/statsd.go:476 +0x64
xxxx/vendor/github.com/DataDog/datadog-go/statsd.(*Client).Count(0xd282a0, 0x57161e, 0x16, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x3ff00000, ...)
        xxxxxxxxxxx/vendor/github.com/DataDog/datadog-go/statsd/statsd.go:491 +0xb0
xxxx/vendor/github.com/DataDog/datadog-go/statsd.(*Client).Incr(0xd282a0, 0x57161e, 0x16, 0x0, 0x0, 0x0, 0x0, 0x3ff00000, 0xa6cb4210, 0x4582c0)
        xxxxxxxxxxx/vendor/github.com/DataDog/datadog-go/statsd/statsd.go:511 +0x5c
xxxx/vendor/gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer.newTracer(0xcc1de4, 0x1, 0x1, 0x6f)
        xxxxxxxxxxx/vendor/gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer/tracer.go:175 +0x7c
xxxx/vendor/gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer.Start(0xcc1de4, 0x1, 0x1)
        xxxxxxxxxxx/vendor/gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer/tracer.go:96 +0x40
main.main()
        xxxxxxxxxxx
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,9 @@
 language: go
 
+arch:
+  - amd64
+  - arm64
+
 go:
   - 1.8
   - 1.9
@@ -9,7 +13,20 @@ go:
   - 1.13.x
   - 1.14.x
 
+# test 32bit build too.
+jobs:
+  include:
+    - arch: amd64
+      go: 1.14.x
+      script:
+        - GOARCH=386 go test -v ./...
+    - arch: amd64
+      go: 1.8
+      script:
+        - GOARCH=386 go test -v ./...
+
 script:
- - go test -race -v ./...
+  # race detector is only available on amd64
+ - if [[ "$TRAVIS_CPU_ARCH" == "amd64" ]]; then go test -race -v ./...; else go test -v ./...; fi
  - go vet ./...
  - "[ -z \"`go fmt ./...`\" ]"

--- a/statsd/sender.go
+++ b/statsd/sender.go
@@ -31,7 +31,7 @@ type sender struct {
 	transport statsdWriter
 	pool      *bufferPool
 	queue     chan *statsdBuffer
-	metrics   SenderMetrics
+	metrics   *SenderMetrics
 	stop      chan struct{}
 }
 
@@ -40,6 +40,7 @@ func newSender(transport statsdWriter, queueSize int, pool *bufferPool) *sender 
 		transport: transport,
 		pool:      pool,
 		queue:     make(chan *statsdBuffer, queueSize),
+		metrics:   &SenderMetrics{},
 		stop:      make(chan struct{}),
 	}
 

--- a/statsd/statsd.go
+++ b/statsd/statsd.go
@@ -209,7 +209,7 @@ type Client struct {
 	flushTime     time.Duration
 	bufferPool    *bufferPool
 	buffer        *statsdBuffer
-	metrics       ClientMetrics
+	metrics       *ClientMetrics
 	telemetryTags []string
 	stop          chan struct{}
 	wg            sync.WaitGroup
@@ -307,6 +307,7 @@ func newWithWriter(w statsdWriter, o *Options, writerName string) (*Client, erro
 	c := Client{
 		Namespace: o.Namespace,
 		Tags:      o.Tags,
+		metrics:   &ClientMetrics{},
 	}
 	if o.Aggregation {
 		c.agg = newAggregator(&c)

